components/chip/chip.vue

```diff
@@ -45,15 +45,18 @@
       @click="$emit('close')"
     >
       <template #icon>
-        <icon-close />
+        <dt-icon
+          name="close"
+          size="300"
```

Comment:
Conditional for Chip icon size, for both left icon slot and close icon:

```
CHIP SIZE
XS → icon size 200
SM → icon size 300 
MD → icon size 300  
```

[Figma Chip component reference](https://www.figma.com/file/2adf7JhZOncRyjYiy2joil/DT-Core%3A-Components-7?node-id=9937%3A64803&viewport=1039%2C656%2C0.33&t=YTNtoWk17389qmkE-11)